CHANGELOG.md

```diff
@@ -19,3 +19,12 @@ This project honours [Semantic Versioning](http://semver.org/).
 Initial release of pan-and-zoom component.
 
 [v1.0.0]: https://github.com/Alhadis/PanAndZoom/releases/tag/v1.0.0
+
+
+
+[v1.0.1]
+------------------------------------------------------------------------
+**September 4th, 2022**  
+Rewritten to TypeScript. Contributed by [Anton Piliugin](https://github.com/piliugin-anton)
+
+[v1.0.1]: https://github.com/Alhadis/PanAndZoom/releases/tag/v1.0.1
```

Comment:
This should go under `[Unpublished]` instead. 😉

Only upon release is `[Unpublished]` replaced with the name of the new tag, a date added under the divider-line, and a link to the release added:

~~~diff
-[Unpublished]: ../../compare/v1.0.0...HEAD
+[Unpublished]: ../../compare/v1.0.1...HEAD
 
-[Unpublished]
+[v1.0.1]
 ------------------------------------------------------------------------
+**September 5th, 2022**
 * **Fixed:** Breakage when loading non-ESM version in browsers
 * **Fixed:** Values not clamped or typecast if set during initialisation
+
+[v1.0.1]: https://github.com/Alhadis/PanAndZoom/releases/tag/v1.0.1
~~~

That's just how I prefer to format change-logs (with newer releases appearing at the top, and older releases at the bottom). You're free to use whichever format you like most once/if this repository's traded hands. 😉